app/src/main/res/values/strings.xml

```diff
@@ -793,6 +793,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_split_way_tutorial">Tap on the way to split it at the point(s) where the answer differs. Try to be as precise as possible, you can zoom in as usual.</string>
     <string name="quest_split_way_too_imprecise">"Please zoom in further"</string>
     <string name="quest_split_way_many_splits_confirmation_description">These are quite a few splits. You can always further split up the way later.</string>
+    <string name="quest_generic_answer_is_actually_steps">"There are steps here"</string>
```

Comment:
@peternewman @smichel17 @andrewharvey Any advice on concise wording here?

Reply:
Preferably one encoraging to split first - rather than turning into steps footway which is steps only in a small part.

Reply:
Is that better?

```suggestion
    <string name="quest_generic_answer_is_actually_steps">"It’s a staircase or step"</string>
```

Reply:
IMO not, because I think staircase is most of all a "Treppenhaus" and step is "eine Stufe".

How about
- "It's actually steps"
- "This part are steps"

Reply:
Or "these are steps"

German translation would be "Das ist eine Treppe"

Reply:
Hmm, I don't really like the mixture of singular and plural.

What about "Consists only of steps"?

Reply:
I changed it to "These are steps" in [`2a7212a` (#3334)](https://github.com/streetcomplete/StreetComplete/pull/3334/commits/2a7212af4f1e31d91a6219c8ef9df16102aefbc2).